HermesProxy/World/Client/PacketHandlers/SpellHandler.cs

```diff
@@ -126,6 +126,13 @@ void HandleUnlearnedSpells(WorldPacket packet)
         [PacketHandler(Opcode.SMSG_CAST_FAILED)]
         void HandleCastFailed(WorldPacket packet)
         {
+            // Artificial lag is needed for spell packets,
+            // or spells will bug out and glow if spammed.
+            if (Settings.ClientSpellDelay > 0)
+            {
+                Thread.Sleep(Settings.ClientSpellDelay);
+            }
```

Comment:
Make just in two lines to reduce the total number of lines in each file
```cs
if (Settings.ClientSpellDelay > 0)
    Thread.Sleep(Settings.ClientSpellDelay);
```
(Applicable for all other delays)